workloads/kube-burner/workloads/large-networkpolicy-egress/README.md

```diff
@@ -0,0 +1,30 @@
+
+## large-networkpolicy-egress
+
+With the help of [large-networkpolicy-egress] customer cases that combined with node-density-heavy, network policy and egress firewall
```

Comment:
@liqcui  This sentence seems not completed.

Reply:
@qiliRedHat Thank you for taking time to review my pr, I have updated the README, please review again, if still have something that didn't describe clearly. thanks!